lib/benchee/formatters/console.ex

```diff
@@ -58,17 +67,19 @@ defmodule Benchee.Formatters.Console do
 
   """
   @spec format(Suite.t) :: [any]
-  def format(%Suite{statistics: jobs_per_input,
+  def format(%Suite{scenarios: scenarios,
                     configuration: %{formatter_options: %{console: config}}}) do
-    Enum.map(jobs_per_input, fn({input, jobs_stats}) ->
-      [input_header(input) | format_jobs(jobs_stats, config)]
-    end)
+    scenarios
+    |> Enum.group_by(&(&1.input_name))
```

Comment:
Of course people that are as smart as I am can _easy_ tell how much better shorter syntax is!
<img width="399" alt="image-1" src="https://user-images.githubusercontent.com/8422484/28280532-4d325242-6b24-11e7-9141-9dd1939cc1d5.png">

Joking aside, I don't really have strong feelings either way. I'll update those tomorrow when I play around with the scenario context stuff.